### ISSUE TITLE
Type usages are ignored in let-expression

### ISSUE THREAD

**Issue by stil4m** [mbuscemi/elm-lens#14]
For example in the following file:

```
module Test exposing (foo)


type alias Thing =
    { x : Int
    }


foo : Int -> Int
foo x =
    let
        f : Thing
        f =
            { x = x }
    in
    f.x
```

`Thing` is annotated with 0 internal references, while it is being used within a let expression in `foo`.

I'm using version `0.1.2` of elm-lens.

**Comment by mbuscemi** [comment]
@stil4m Can you please confirm that this is fixed for you in v0.2.0?

**Comment by stil4m** [comment]
Tested in 0.2.0 and it is fixed 👍
